modules/test/playwright/tests/account-admin-web/env/portal-ext.properties

```diff
@@ -1 +1,3 @@
-company.security.strangers.verify=false
\ No newline at end of file
+company.security.strangers.verify=false
+
+captcha.enforce.disabled=true
```

Comment:
remove empty space between the two lines and sort them

Reply:
Hi @smottal, SF resets it to this when I run it. Should I just ignore it?

Reply:
yes, if it's a SF rules leave it as it is